src/pages/Timeline.js

```diff
@@ -180,58 +181,59 @@ const Timeline = ({
-
-  const MCD_SOURCE = '0xF44113760c4f70aFeEb412C63bC713B13E6e202E';
-
   return (
     <Fragment>
       {DEV_USE_MIGRATION_BANNER ? <MigrationNotificationBanner /> : null}
       <VoterStatus signaling={signaling} legacy={true} />
       <RiseUp key={otherProposals.toString()}>
-        {signaling || !hatProposal ? null : (
+        {signaling || !governingProposal ? null : (
           <StyledCard>
             <Card.Element height={proposalWrapperHeight}>
               <ProposalDetails>
                 <div style={{ display: 'flex' }}>
                   <ExtendedLink
-                    to={`/executive-proposal/${toSlug(hatProposal.title)}`}
+                    to={`/executive-proposal/${toSlug(
+                      governingProposal.title
+                    )}`}
                   >
-                    <SubHeading>{hatProposal.title}</SubHeading>
+                    <SubHeading>{governingProposal.title}</SubHeading>
                   </ExtendedLink>
-                  <Tag ml="16" green>
-                    GOVERNING PROPOSAL
-                  </Tag>
                 </div>
-                <Body
-                  dangerouslySetInnerHTML={{
-                    __html: hatProposal.proposal_blurb
-                  }}
-                />
-                <ExtendedLink
-                  to={`/executive-proposal/${toSlug(hatProposal.title)}`}
-                >
-                  Read more...
-                </ExtendedLink>
-                <div>
-                  {!!hatProposal.end_approvals ? (
-                    <Tag>{`Executed on ${formatDate(
-                      hatProposal.end_timestamp
-                    )} with ${formatRound(
-                      hatProposal.end_approvals
-                    )} MKR`}</Tag>
-                  ) : (
-                    <div>
-                      <Tag>
-                        {hatProposal.source === MCD_SOURCE
-                          ? 'Available for execution on November 18th at 16:00 UTC'
-                          : 'Available for execution'}
-                      </Tag>
-                    </div>
-                  )}
+                <Body>
+                  {governingProposal.proposal_blurb}.
+                  <ExtendedLink
+                    to={`/executive-proposal/${toSlug(
+                      governingProposal.title
+                    )}`}
+                  >
+                    {' Read more.'}
+                  </ExtendedLink>
+                </Body>
+                <div style={{ display: 'flex' }}>
+                  <Tag mr="16" green>
+                    Governing Proposal
+                  </Tag>
+                  <Tag>{`Passed on ${formatDateWithTime(
+                    governingProposal.datePassed
+                  )}${
+                    governingProposal.end_approvals
+                      ? ' with ' +
+                        formatRound(governingProposal.end_approvals) +
```

Comment:
should this be bold?

Reply:
Yeah, but do you have any idea why the <b> tag wouldn't do anything? If I use <i> it italicizes, but <b> does nothing

Reply:
hmm interesting, my only guess would be that the `font-weight` being set by the `<b>` tag is somehow being overridden by a parent